### PR TITLE
Add a user manual for configuration management

### LINKED ISSUE #316: Added the user guide for configuration management.
### What would you like to be added?

1、Added the user guide for configuration management.

### Why is this needed?

1、The configuration management function is added to the backend, but the user guide is not provided.

### PR DESCRIPTION
**Pull request by hanbingleixue**
What type of PR is this?

doc

What this PR does / why we need it?

1、Add a user manual for configuration management

Which issue(s) this PR fixes:

Fixes #316

### CODE REVIEW

File: docs/zh/document/user-guide/sermant-backend.md [hunk 1]
```diff
@@ -1,150 +1,271 @@
...
-> 注：Backend为**非必要组件**，用户可按需部署。
+Sermant Backend与配置中心配合使用，可以管理配置中心的所有配置项，可以查看、新增、修改、删除配置项。
+
+> 注：Sermant Backend为**非必要组件**，用户可按需部署。
 
 ## 参数配置
+
+### Sermant Agent参数配置
+
+Sermant Backend提供的能力，需要依赖Sermant Agent上报的数据，所以在使用Sermant Backend时，需要先配置Sermant Agent中用于连接Sermant Backend、开启数据上报的相关参数，在配置文件`agent/config/config.properties`中修改以下配置：
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: Sermant Backend提供xxx、xxx能力

File: docs/zh/document/user-guide/sermant-backend.md [hunk 2]
```diff
@@ -1,150 +1,271 @@
...
-- 根据实际环境设置Gateway参数
+| **参数键**         | **说明**                             | **默认值** | **是否必须** |
+| ------------------ | ------------------------------------ | ---------- | ------------ |
+| dynamic.config.enable    | 配置管理的开关 | true     | 否           |
+| dynamic.config.namespace    | 默认的命名空间（连接Nacos配置中心时使用） | default     | 连接配置中心Nacos时必须          |
+| dynamic.config.timeout    | 请求配置中心的超时时间 | 30000     | 置管理的开关开启时必须          |
+| dynamic.config.serverAddress    | 配置中心的连接地址 | 127.0.0.1:30110     |    配置管理的开关开启时必须       |
+| dynamic.config.dynamicConfigType    | 配置中心的类型，支持ZOOKEEPER、NACOS、KIE | KIE     |    配置管理的开关开启时必须       |
+| dynamic.config.connectTimeout    |连接配置中心的超时时间 | 3000     |    配置管理的开关开启时必须       |
+| dynamic.config.enableAuth    |是否开启授权认证，支持Nacos和Zookeeper） | false     |    配置管理的开关开启时必须       |
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: unnecessary  `)`

File: docs/zh/document/user-guide/sermant-backend.md [hunk 3]
```diff
@@ -1,150 +1,271 @@
...
+| dynamic.config.userName    |授权认证时使用的用户名（明文） | null     |    开启授权认证时必须       |
+| dynamic.config.password    |授权认证时使用的密码（密文） | null     |    开启授权认证时必须       |
+| dynamic.config.secretKey    |密码加密时使用的密钥 | null     |    开启授权认证时必须       |
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: 密码采用xxx方式加密时使用的密钥

File: docs/zh/document/user-guide/sermant-backend.md [hunk 4]
```diff
@@ -1,150 +1,271 @@
...
+    <MyImage src="/docs-img/backend/backend-webhook-feishu.png"></MyImage>
+
+  - 钉钉测试事件推送如下图所示：
+
+    <MyImage src="/docs-img/backend/backend-webhook-dingding.png"></MyImage>
+
+### 6. 配置管理验证
+
+- 通过浏览器访问地址`http://127.0.0.1:8900/`
+- 点击菜单栏**配置管理**进入配置管理界面既可查看配置中心的配置，配置管理页面默认会展示路由插件的配置，如下图所示:
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: 即可

File: docs/zh/document/user-guide/sermant-backend.md [hunk 5]
```diff
@@ -1,150 +1,271 @@
...
+
+- 在配置管理页面可以进行配置的删除操作，点击删除按钮会提示是否进行删除操作。如下图所示：
+
+<MyImage src="/docs-img/backend/backend-config-delete.png"></MyImage>
+
+- 选择**是**之后会删除该配置项，如下图所示：
+
+<MyImage src="/docs-img/backend/backend-config-delete-result.png"></MyImage>
+
+#### 6.3. 验证配置新增
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: should be 6.1

File: docs/zh/document/user-guide/sermant-backend.md [hunk 6]
```diff
@@ -1,150 +1,271 @@
...
+
+<MyImage src="/docs-img/backend/backend-config-add-info.png"></MyImage>
+
+- 配置新增成功之后，即可在配置管理页面查询到该配置项，如下图所示：
+
+<MyImage src="/docs-img/backend/backend-config-add-result.png"></MyImage>
+
+#### 6.4. 验证配置修改
+
+- 在配置管理页面点击查看按钮既可以跳转到配置详情页面。如下图所示：
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: 即可

File: docs/zh/document/user-guide/sermant-backend.md (hunk 1 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

File: docs/zh/document/user-guide/sermant-backend.md (hunk 2 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

File: docs/zh/document/user-guide/sermant-backend.md (hunk 3 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

File: docs/zh/document/user-guide/sermant-backend.md (hunk 4 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

File: docs/zh/document/user-guide/sermant-backend.md (hunk 5 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

File: docs/zh/document/user-guide/sermant-backend.md (hunk 6 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

**Pull request merged**